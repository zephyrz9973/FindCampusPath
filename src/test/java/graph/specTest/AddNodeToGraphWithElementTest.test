# The test is for add node to graph with element already
# Create a graph
CreateGraph graph1

# Add a pair of nodes
AddNode graph1 n1
AddNode graph1 n2

# Add an edge
AddEdge graph1 n1 n2 e1

# Print the nodes in the graph and the outgoing edges from n1
ListNodes graph1
ListChildren graph1 n1