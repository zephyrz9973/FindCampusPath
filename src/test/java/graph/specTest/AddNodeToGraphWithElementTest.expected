# The test is for add node to graph with element already
# Create a graph
created graph graph1

# Add a pair of nodes
added node n1 to graph1
added node n2 to graph1

# Add an edge
added edge e1 from n1 to n2 in graph1

# Print the nodes in the graph and the outgoing edges from n1
graph1 contains: n1 n2
the children of n1 in graph1 are: n2(e1)